### ISSUE TITLE
Feature Request - After adding an image file to your project, open it in the editor.

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#2316]
Right now, it gets added to the file tree, but the view doesn't switch away from what you were currently doing. I think it would be nice to switch to, as a confirmation that a) it worked and b) that you uploaded the right image.

**Comment by flukeout** [comment]
We could add some sort of callback to the image resizing code to do this maybe?

**Comment by humphd** [comment]
We used to do this, and I think I removed it.  It's certainly doable.  There's a [function in the dnd code for doing it](https://github.com/mozilla/brackets/blob/master/src/utils/DragAndDrop.js#L137), we just need to call it when we're done importing.

**Comment by flukeout** [comment]
Do you remember why we removed it @humphd

**Comment by humphd** [comment]
I honestly can't remember.  I think it might have been an optimization for when you drag and drop a ton of files in, and it tries to open them all at once.  But I can't remember all the details.  It's not hard to put back in if you want it.

**Comment by flukeout** [comment]
Yeah, that's a good point, makes sense to just open it at the end of a multi-file import on whatever image we finish on.

**Comment by flukeout** [comment]
I think one-at-a-time imports will still be our default mode for this feature though.
